mapk_demon/transmitters.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import division
+
+from functools import partial
+
+from pyro import sample
+from pyro.distributions import LogNormal
+from torch import tensor
+
+
+def g1(a):
+    return a / (a + 1)
+
+
+def g2(a):
+    return a**2 / (a**2 + a + 1)
+
+
+def f(mu, N):
+    return N ** (0.5) * mu + mu
```

Comment:
This should be `N * mu ** 0.5 +  mu`.  This way the mean is mu and the variance is mu.